### ISSUE TITLE
distinguished_name as described not working

### ISSUE THREAD

**Issue by tasptz** [uilianries/conan-ldap-authentication#11]
The config

`distinguished_name: cn=$username,ou=Users,dc=company,dc=org`

did not work for me. Instead I had to use

`distinguished_name: $username@company.org`

I don't know if this is an issue of the plugin, python ldap library or the ldap server itself.
Maybe it would be possible to provide some insight in the plugin documentation?

**Comment by uilianries** [comment]
Yeah, you are right. LDAP is very flexible to use distinguished fields.

I need to improve the readme and add more fields support.
